<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchUi</class>
 <widget class="QWidget" name="SearchUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>135</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="findPrevPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Find previous occurrence (Shift+Enter)</string>
       </property>
       <property name="text">
        <string>&amp;&lt; Find</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="replaceallPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Replace all occurrences</string>
       </property>
       <property name="text">
        <string>Replace &amp;All</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="findCO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>666</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="minimizePB">
       <property name="toolTip">
        <string>Hide replace and option widgets</string>
       </property>
       <property name="text">
        <string>&amp;Minimize</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="replaceLA">
       <property name="text">
        <string>Rep&amp;lace with:</string>
       </property>
       <property name="buddy">
        <cstring>replaceCO</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="findLA">
       <property name="text">
        <string>&amp;Search:</string>
       </property>
       <property name="buddy">
        <cstring>findCO</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="replaceCO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>666</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="replacePB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Replace and find next occurrence</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>&amp;Replace &gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="replacePrevPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Replace and find previous occurrence</string>
       </property>
       <property name="text">
        <string>&lt; Re&amp;place</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="findPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Find next occurrence (Enter)</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Find &amp;&gt;</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="CBFrame">
     <layout class="QHBoxLayout" name="CBLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="caseCB">
          <property name="toolTip">
           <string>Treat uppercase/lowercase writing as distinct</string>
          </property>
          <property name="text">
           <string>&amp;Case sensitive[[search]]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wordsCB">
          <property name="toolTip">
           <string>Match whole words only</string>
          </property>
          <property name="text">
           <string>Wh&amp;ole words</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="selectionCB">
          <property name="toolTip">
           <string>Limit search and replace to selection</string>
          </property>
          <property name="text">
           <string>Selection onl&amp;y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="instantSearchCB">
          <property name="toolTip">
           <string>If this is checked, LyX will search forward immediately</string>
          </property>
          <property name="text">
           <string>Search as yo&amp;u type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wrapCB">
          <property name="toolTip">
           <string>If the end/start of the document is reached, automatically wrap around</string>
          </property>
          <property name="text">
           <string>&amp;Wrap</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
